index.d.ts

```diff
@@ -1,21 +1,25 @@
-import { Plugin } from "fastify";
-import { Server, IncomingMessage, ServerResponse } from "http";
-import { Http2SecureServer, Http2Server, Http2ServerRequest, Http2ServerResponse } from "http2";
-import * as https from "https";
-
-type HttpServer = Server | Http2Server | Http2SecureServer | https.Server;
-type HttpRequest = IncomingMessage | Http2ServerRequest;
-type HttpResponse = ServerResponse | Http2ServerResponse;
-
-declare module 'fastify' {
-    interface FastifyInstance<
-        HttpServer,
-        HttpRequest,
-        HttpResponse
-        > {
-        blipp: () => void;
-    }   
-}
-declare function fastifyBlipp(): Plugin<HttpServer, HttpRequest, HttpResponse, any>;
-declare namespace fastifyBlipp {}
-export = fastifyBlipp;
+import { Plugin } from "fastify";
```

Comment:
I believe in this file the formatting was changed (or what?) so the diff is bigger, that it is supposed to be.
Can you, please, have a look?

Reply:
Oh, seems like it, I'll fix that and update examples